docs/source/scancode-toolkit/how_to/how_to_run_a_scan.rst

```diff
@@ -6,46 +6,56 @@ How to Run a Scan
 Quickstart
 ----------
 
-ScanCode results are provided as:
-
 
-#. JSON file (default)
-#. html (static html)
+ScanCode results are provided as:
 
-The basic usage is:
+1. JSON file (default)
+2. html (static html)
+3. csv
 
-::
+The basic usage is::
 
-   ./scancode [OPTIONS] <input> <output_file>
+    ./scancode [OPTIONS] <OUTPUT FORMAT(S)> <input>
```

Comment:
Hi, @RituRajSingh878 the [options] include setting output formats as you can see by running `./scancode --help`. You need not specify that explicitly. Please discuss what errors you are getting in the Gitter channel. Everyone will help you out.